src/services/wfspermalink.js

```diff
@@ -0,0 +1,304 @@
+goog.provide('ngeo.WfsPermalink');
+
+goog.require('ngeo');
+/** @suppress {extraRequire} - required for `ngeoQueryResult` */
+goog.require('ngeo.Query');
+goog.require('ol.format.WFS');
+
```

Comment:
mhm no, doesn't fix the error. Do I maybe need to add `@export` to the namespace in OpenLayers?